paddlespeech/audio/utils/tensor_utils.py

```diff
@@ -248,7 +248,7 @@ def st_reverse_pad_list(ys_pad: paddle.Tensor,
     #   >>> tensor([[ 2,  1,  0],
     #   >>>         [ 2,  1,  0],
     #   >>>         [ 0, -1, -2]])
-    index = index * seq_mask.astype(index.dtype)
+    index = index * paddle.cast(seq_mask, 'int64')
```

Comment:
suggested use `seq_mask.astype(index.dtype)`  instead cast seq_mask to int64

Reply:
OK